src/nidmm/templates/functions.rst/get_cal_date_and_time.rst.mako

```diff
@@ -1,33 +0,0 @@
-<%page args="function, config, method_template, indent"/>\
```

Comment:
How is this related to session lock? Merge issue?

Reply:
Unused file. Should I not clean up when I see an issue? Seems overkill to create a PR just to delete unused files

Reply:
> Unused file. Should I not clean up when I see an issue? Seems overkill to create a PR just to delete unused files

The reviewer of a 67 file PR might disagree. Regardless, it wasn't clear to me if this was accidental or not. The PR's description didn't call it out either.

Reply:
We're down to 55 files changed!